ow_faults/src/FaultInjector.cpp

```diff
@@ -12,8 +12,9 @@ using namespace ow_lander;
 
 FaultInjector::FaultInjector(ros::NodeHandle node_handle)
 {
-  m_joint_state_sub = node_handle.subscribe("/joint_states", 10, &FaultInjector::jointStateCb, this);
-  m_joint_state_pub = node_handle.advertise<sensor_msgs::JointState>("/faults/joint_states", 10);
+  m_joint_state_sub = node_handle.subscribe("/orig_joint_states", 10, &FaultInjector::jointStateCb, this);
```

Comment:
Please make it /original/joint_states. This is more consistent with other ROS topic naming/categorization. And there's really no need to abbreviate "original."

Reply:
I agree with Terry. I would even **suggest** (not a strong opinion) to use `/_original/joint_states` if that's possible to indicate that this is topic is for private use. But I don't know if prefixing a topic name with underscore is permissible.

Reply:
will do!